src/cc_helpers.cc

```diff
@@ -1626,7 +1611,8 @@ bool secure_authenticated_channel::init_client_ssl(const string& host_name, int
   int ret = SSL_connect(ssl_);
   if (ret <= 0) {
     int err = SSL_get_error(ssl_, ret);
-    printf("init_client_ssl: ssl_connect failed, err: %d\n", err);
+    printf("init_client_ssl: ssl_connect failed, ret=%d, err=%d: %s\n",
+           ret, err, ssl_strerror(err));
```

Comment:
Enhanced to now use the new `ssl_strerror()` to print the `err` value as a text string. Helps debugging.

Reply:
good